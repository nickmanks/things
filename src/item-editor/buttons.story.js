/* global module */
import React from 'react';
import {storiesOf} from '@storybook/react';
import EditorButtons from './buttons';
import testItems from '../testing/test-items';


storiesOf('Simple/EditorButtons', module)
  .add('with multiple items', ()=> (
    <EditorButtons
      selected={testItems['test-id-1']}
      onCloseModal={()=> null}
      onArchive={()=> null}
    />
  ));
