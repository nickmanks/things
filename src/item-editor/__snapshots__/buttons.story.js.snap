// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Simple/EditorButtons with multiple items 1`] = `
<ModalFooter>
  <div className={"modal-footer"}>
    <Button id={"archive"} pill={true} outline={true} theme={"primary"} onClick={"function"} tag={"button"}>
      <button type={"button"} id={"archive"} onClick={"function"} className={"btn btn-outline-primary btn-pill"}>
        Archive
      </button>
    </Button>
    <Button id={"done"} pill={true} theme={"primary"} onClick={"function"} tag={"button"}>
      <button type={"button"} id={"done"} onClick={"function"} className={"btn btn-primary btn-pill"}>
        Done
      </button>
    </Button>
  </div>
</ModalFooter>
`;
